src/node.h

```diff
@@ -287,8 +287,11 @@ NODE_EXTERN struct uv_loop_s* GetCurrentEventLoop(v8::Isolate* isolate);
 /* Converts a unixtime to V8 Date */
 NODE_DEPRECATED("Use v8::Date::New() directly",
                 inline v8::Local<v8::Value> NODE_UNIXTIME_V8(double time) {
-  return v8::Date::New(v8::Isolate::GetCurrent(), 1000 * time);
-})
+                  return v8::Date::New(
```

Comment:
Nit: Is the indentation off?

Reply:
This was done by `make format-cpp`

Reply:
Is it worth it?
It's already NODE_DEPRECATED, so IMHO when V8 drops this, we can drop it too.

Reply:
@refack are you talking about the change itself? I did it to avoid a compilation warning with V8 7.1

Reply:
Ok. I just saw the compilation output.